.rubocop.yml

```diff
@@ -37,3 +37,6 @@ Style/RescueModifier:
 
 Style/WordArray:
   Enabled: false
+
+Style/SingleLineBlockParams:
```

Comment:
Rubocop actually forced reduce operations to use var names 'a' and 'e' in a block, as in: 
```ruby
(...).reduce(0) { |a, e| (...)
```
This rule has been invalidated in the ruby style guide and also disabled in future versions of rubocop, so disabling this here for improved readability.